tests/unit/test_solvers/test_processed_variable.py

```diff
@@ -43,10 +47,14 @@ def test_processed_variable_1D(self):
         t_sol = np.linspace(0, 1)
         y_sol = np.ones_like(x_sol)[:, np.newaxis] * np.linspace(0, 5)
 
-        processed_var = pybamm.ProcessedVariable(var_sol, pybamm.Solution(t_sol, y_sol))
+        processed_var = pybamm.ProcessedVariable(
+            var_sol, pybamm.Solution(t_sol, y_sol), warn=False
+        )
```

Comment:
Oh, I see, solution doesn't have a model here for timescale. Maybe `Solution.model` could be initialized to `BaseModel`?

Reply:
yep, this worked thanks!